### PR TITLE
added the geolocationPositionErrorInstance.message to the code as it seems that it should display the error message to the user

### PR DESCRIPTION
**Pull request by Serhazor**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [ ] This branch is rebased/merged with the **latest master**.
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/utils/geolocation.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const asyncGetGeoLocation = () => {
       resolve({ lat: latitude, lng: longitude });
     };
     const onGeoError = () => {
```
[the remark this answers is not in the document]
Comment (Serhazor) [reply client/src/utils/geolocation.js:9]: got a "little" stuck there with couple of things. First and most important I guess is that I have no idea how to reach out to you on Slack ?

[the remark this answers is not in the document]
Comment (joshmorel) [reply client/src/utils/geolocation.js:9]: > First and most important I guess is that I have no idea how to reach out to you on Slack ?

Have you onboarded with the FightPandemics team (I don't see a Sergej)? https://www.volunteerconnector.org/dover/fightpandemics

Certainly external contributions are welcome but at this stage in our project most have onboarded as volunteers, joined the slack etc..

[the remark this answers is not in the document]
Comment (Serhazor) [reply client/src/utils/geolocation.js:9]: Now that's gonna leave a mark (c) 

No I did not. Can I still join as a volunteer? Would be more then happy to get involved.

[the remark this answers is not in the document]
Comment (Serhazor) [reply client/src/utils/geolocation.js:9]: Applied :-)

**Comment by joshmorel** [comment]
Haven't heard anything so will have someone else start new on this one.

**Comment by Serhazor** [comment]
> Haven't heard anything so will have someone else start new on this one.

I've applied to volunteer for your organisation, but did not hear back yet.

**Comment by joshmorel** [comment]
Ah okay @Serhazor . I'll reach out to someone. Once you get in I'll find another issue to assign.

**Comment by joshmorel** [comment]
@Serhazor  They couldn't see for you in...

Can you try through http://fightpandemics.workable.com

Thanks.

**Comment by Serhazor** [comment]
> @Serhazor They couldn't see for you in...
> 
> Can you try through http://fightpandemics.workable.com
> 
> Thanks.

That's the link I've applied through on 24th of July (my email is I applied with is sergej.dikun@gmail.com )

**Pull request closed without merging**